### PR TITLE
Bump version to preview4

### PR DESCRIPTION
**Pull request by prafullbhosale**
This is LTS version with Project.json support. 

cc @natemcmaster @pranavkm @NTaylorMullen

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
Why the code changes?

**Comment by prafullbhosale** [comment]
The ProjectJsonWorkspaces code was duplicated in this repo due to a restore issue long long time back. 
The code is since outdated and has some bug fixes in that. 
So removed the duplicated code and took a dependency on the ProjectModel.Workspaces package.

**Comment by NTaylorMullen** [comment]
Will it work without the code changes? 

There were some bug fixes done by RazorTooling that Damian suggest we not take to ensure no bugs. Slightly scary to have code changes when the goal is to just re-ship old bits with new versions.

**Comment by NTaylorMullen** [comment]
Also note that the bug fixes done by Razor were super minor which is why it never resulted in an update.

**Comment by prafullbhosale** [comment]
Yes, it would work without the code changes. Will revert the code changes and update this.

**Comment by prafullbhosale** [comment]
:up: :date:

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json [hunk 1]
```diff
@@ -6,21 +6,16 @@
...
     "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0",
     "Microsoft.DotNet.ProjectModel": "1.0.0-rc3-003121",
-    "Microsoft.Extensions.PlatformAbstractions": "1.0.0"
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0",
+    "NETStandard.Library": "1.6.0"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Threading.Tasks": {
```
Comment (NTaylorMullen) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json]: This is definitely weird that it was here before howeeevvver, same question as the code changes before but with the dependency changes here being needed.

**Comment by NTaylorMullen** [comment]
Last question is this: https://github.com/aspnet/Scaffolding/pull/328#discussion_r86494754. If they're not needed / don't solve any significant issue I'd suggest undoing the change; otherwise, if it can't work properly without then keep em in.

Other then that, :shipit:

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json (hunk 1 above)
Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json]: Keeping this here and not adding NETStandard.Library 1.6.0 was causing build failure on my local about a missing System.* assembly reference for net451

**Pull request merged**